Search for parameter

Alexis09Luque opened the pull request:
Búsqueda general por parámetro, por favor indicar correciones

Alexis09Luque committed:
feat: First Creating about search for parameters

Se creo un método para relizar la búsqueda por parámetro donde se busca en los campos name,surname  y dni, aún se espera optimizar este código

Alexis09Luque committed:
fix(Employee Controller): Validation about field page

se añadio la validacióm para el atributo page en el request del API para búsqueda e empleado y ayuda a resolver : #27

Alexis09Luque committed:
feat: Search to parameter general

Se creo la API para buscar por parámetro de manera general y se realizo avances en la búsqueda por filtro esto ayuda a resolver : #27

locutermo commented on routes/web.php:
```diff
@@ -12,6 +12,7 @@
 */
 
 $router->get('/employees', 'EmployeeController@index');
+$router->get('/search', 'EmployeeController@search');
```
Esta busqueda es para empleados asi que falta agregar /employees antes de search

Alexis09Luque committed:
fix:Correction about search

 Se modifico en método search en el controlador y se modifico la ruta y resuelve : #27

locutermo reviewed (commented):
Revisar correccion
